core/frontend/src/components/vehiclesetup/configuration/failsafes/Failsafes.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div v-if="params_finished_loaded" class="pt-1 pb-1">
+    <v-sheet class="ma-3 pa-1 d-flex flex-row flex-wrap flex-grow-0 justify-center">
+      <failsafe-card
+        v-for="failsafeDefinition in failsafes"
+        :key="failsafeDefinition.name"
+        :failsafe-definition="failsafeDefinition"
+      />
+    </v-sheet>
+  </div>
+</template>
+
+<script lang="ts">
+import failsafeCard from '@/components/vehiclesetup/configuration/failsafes/FailsafeCard.vue'
+import autopilot_data from '@/store/autopilot'
+
+import { FailsafeDefinition } from './types'
+
+export default {
+  name: 'FailsafesConfigration',
+  components: {
+    failsafeCard,
+  },
+  data() {
+    return {
+      failsafes: [] as FailsafeDefinition[],
+    }
+  },
+  computed: {
+    params_finished_loaded(): boolean {
+      return autopilot_data.finished_loading
+    },
+  },
+  mounted() {
+    // we fetch the failsafes on mounting because we
+    // need to be in an async context to wait for the import calls
+    this.get_failsafes().then((failsafes) => {
+      this.failsafes = failsafes
+    })
+  },
+  methods: {
+    async get_failsafes(): Promise<FailsafeDefinition[]> {
+      const failsafes: FailsafeDefinition[] = [
+        {
+          name: 'Ground Station Heartbeat Loss',
+          generalDescription: 'The heartbeat failsafe is triggered when the vehicle does not receive '
+          + 'a heartbeat from the GCS within 3 seconds.',
+          image: (await import('@/assets/img/configuration/failsafes/heartbeat.svg')).default as string,
+
+          params: [
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer',
+              name: 'FS_GCS_TIMEOUT',
+              optional: true,
+            },
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_GCS_ENABLE',
+              replacementDescription: '',
+            },
+          ],
+        },
+        {
+          name: 'Pilot Input Loss',
+          generalDescription: 'The pilot input failsafe is triggered when the vehicle does not receive '
+          + 'any pilot input for a given amount of time.',
+          image: (await import('@/assets/img/configuration/failsafes/pilot-input.svg')).default as string,
+          params: [
+            {
+              replacementTitle: 'Action',
+              icon: 'mdi-lightning-bolt',
+              name: 'FS_PILOT_INPUT',
+            },
+            {
+              replacementTitle: 'Timeout',
+              icon: 'mdi-timer-outline',
+              name: 'FS_PILOT_TIMEOUT',
```

Comment:
```suggestion
              replacementTitle: 'Timeout',
              icon: 'mdi-timer-outline',
              name: 'FS_PILOT_TIMEOUT',
            },
            {
              replacementTitle: 'Action',
              icon: 'mdi-lightning-bolt',
              name: 'FS_PILOT_INPUT',
```
We should be consistent with the ordering.
As a general principle I think the Action should go at the bottom, because everything else has to happen before the action actually triggers.